app/services/proofing/socure/id_plus/proofer.rb

```diff
@@ -0,0 +1,94 @@
+module Proofing
+  module Socure
+    module IdPlus
+      class Proofer
+        attr_reader :config
+
+        VENDOR_NAME = 'socure_kyc'
+
+        VERIFIED_ATTRIBUTE_MAP = {
+          address: %i[streetAddress city state zip].freeze,
+          first_name: :firstName,
+          last_name: :surName,
+          phone: :mobileNumber,
+          ssn: :ssn,
+          dob: :dob,
+        }.freeze
+
+        REQUIRED_ATTRIBUTES = %i[
+          first_name
+          last_name
+          address
+          dob
+          ssn
+        ].to_set.freeze
+
+        # @param [Proofing::Socure::IdPlus::Config] config
+        def initialize(config)
+          @config = config
+        end
+
+        # @param [] applicant
+        # @returns [Proofing::Resolution::Result]
+        def proof(applicant)
+          input = Input.new(applicant)
+
+          request = Request.new(config:, input:)
+
+          response = request.send_request
+
+          build_result_from_response(response)
+        rescue Proofing::TimeoutError, RequestError => err
+          build_result_from_error(err)
+        end
+
+        private
+
+        # @param [Proofing::Socure::IdPlus::Response] response
+        def all_required_attributes_verified?(response)
+          (REQUIRED_ATTRIBUTES - verified_attributes(response)).empty?
+        end
+
+        def build_result_from_error(err)
+          Proofing::Resolution::Result.new(
+            success: false,
+            errors: {},
+            exception: err,
+            vendor_name: VENDOR_NAME,
+            transaction_id: err.respond_to?(:reference_id) ? err.reference_id : nil,
+          )
+        end
+
+        # @param [Proofing::Socure::IdPlus::Response] response
+        # @returns [Proofing::Resolution::Result]
```

Comment:
in YARD it's `@return` singular. The plural version is for JSDoc
```suggestion
        # @return [Proofing::Resolution::Result]
```